pkg/plan/resource/kubeadm_join.go

```diff
@@ -33,6 +33,8 @@ type KubeadmJoin struct {
 	CertificateKey string `structs:"certificateKey" plan:"hide"`
 	// IgnorePreflightErrors is optionally used to skip kubeadm's preflight checks.
 	IgnorePreflightErrors []string `structs:"ignorePreflightErrors"`
+	// External Load Balancer IP address to be used instead of masterIP
+	ExternalLoadBalancer string `structs:"externalLoadBalancer" plan:"hide"`
```

Comment:
I'm curious as to why this is hidden in the plan. One side effect of hiding any attribute is that we won't be able to determine that it has changed when we go to update a node. If we decide at some later time to swap out the load balancer after first standing up a new one, the machine_actuator wouldn't know to reconstruct the node with the new reference (thinking it hadn't changed)...

Reply:
I don't have any particular reason.
I just saw that the `masterIP` field is hidden, so I did the same to this field.

I'll remove the plan:hide from this field.